algebra/src/fields/arithmetic.rs

```diff
@@ -439,18 +437,6 @@ macro_rules! impl_multiplicative_ops_from_ref {
             }
         }
 
-        #[allow(unused_qualifications)]
-        impl<P: $params> MulShort<Self> for $type<P> {
```

Comment:
I do not understand the purpose of the remove code. Was it related to the short Mongomery multiplication?